internal/service/sso/data_source_application.go

```diff
@@ -0,0 +1,1166 @@
+										Computed:    true,
+										NestedObject: schema.NestedAttributeObject{
+											Attributes: map[string]schema.Attribute{
+												"enabled": schema.BoolAttribute{
+													Description: "A boolean that specifies whether device integrity detection takes place on mobile devices.",
+													Computed:    true,
+												},
+												"excluded_platforms": schema.SetAttribute{
+													Description: "Indicates OS excluded from device integrity checking.",
+													ElementType: types.StringType,
+													Computed:    true,
+												},
+												"cache_duration": schema.ListNestedAttribute{
+													Description: "Indicates the caching duration of successful integrity detection calls.",
+													Computed:    true,
+													NestedObject: schema.NestedAttributeObject{
+														Attributes: map[string]schema.Attribute{
+															"amount": schema.Int64Attribute{
+																Description: "An integer that specifies the number of minutes or hours that specify the duration between successful integrity detection calls.",
+																Computed:    true,
+															},
+															"units": schema.StringAttribute{
+																Description: "A string that specifies the cache duration time units.",
+																Computed:    true,
+															},
+														},
+													},
+												},
+												"google_play": schema.ListNestedAttribute{
+													Description: "A single block that describes Google Play Integrity API credential settings for Android device integrity detection.",
+													Computed:    true,
+													NestedObject: schema.NestedAttributeObject{
+														Attributes: map[string]schema.Attribute{
+															"decryption_key": schema.StringAttribute{
+																Description: "Play Integrity verdict decryption key from your Google Play Services account. This parameter must be provided if you have set `verification_type` to `INTERNAL`.  Cannot be set with `service_account_credentials_json`.",
+																Computed:    true,
+															},
+															"service_account_credentials_json": schema.StringAttribute{
+																Description: "Contents of the JSON file that represents your Service Account Credentials.",
+																Computed:    true,
+															},
+															"verification_key": schema.StringAttribute{
+																Description: "Play Integrity verdict signature verification key from your Google Play Services account.",
+																Computed:    true,
+															},
+															"verification_type": schema.StringAttribute{
+																Description: "The type of verification.",
+																Computed:    true,
+															},
+														},
+													},
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+						"bundle_id": schema.StringAttribute{
```

Comment:
This is a deprecated field on the resource, we shouldn't include these in the data source